src/System.IO/tests/TextReader/TextReaderTests.cs

```diff
@@ -0,0 +1,283 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class TextReaderTests
+    {
+        protected virtual char[] GetSmallData()
+        {
+            return new char[] { 'H', 'E', 'L', 'L', 'O' };
+        }
+
+        protected virtual char[] GetLargeData()
+        {
+            char[] testData = new char[] { 'H', 'E', 'L', 'L', 'O' };
```

Comment:
nit, you could write as `var testData = "HELLO".ToCharArray();`

Reply:
Done.,,